src/pybind/mgr/dashboard/controllers/docs.py

```diff
@@ -248,8 +248,8 @@ def _gen_params(cls, params, location):
         return parameters
 
     @classmethod
-    def _gen_paths(cls, all_endpoints, baseUrl):
-        METHOD_ORDER = ['get', 'post', 'put', 'delete']
+    def _gen_paths(cls, all_endpoints, base_url):
+        METHOD_ORDER = ['get', 'post', 'put', 'delete']  # noqa: N806
```

Comment:
I thins case I think pep8-naming does not allow 'constants' within methods, so if it's really a constant should be in an outer scope.

Reply:
You're right, this can simply be fixed. While going through the pep8-naming errors my intention was to change as less code as possible.